lib/src/components/NxTabs/types.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, ReactNode } from 'react';
+import PropTypes from 'prop-types';
+
+export type Props = HTMLAttributes<HTMLDivElement> & {
+  activeTab?: number | null ;
+  onTabSelect?: ((index: number) => void) | null;
```

Comment:
Should this really be optional?  The component doesn't really work without it.

Reply:
Good point, I suppose even if you had other ways of changing the active tab you'd still want the normal tab selection to work.

Reply:
649044c660cc75e52c2b8dc4451b10df15be5f51

I also copied the description of the `onTabSelect` over to the `NxStatefulTabsPage.tsx`.